### PR TITLE
Don't use sales strategies as a fallback for the end date for Zora

### PR DESCRIPTION
**Pull request by agans**
### **User description**
- I don't think these sales strategies are typically relevant in practice at all for Zora mints availability dates
- Eg, the zora_timed_minter one has an end date of `"0"` on these two mints: https://zora.co/collect/zora:0x44ac03c9101257fbd4f489d1ce9adcd466c1c27e/2 + https://zora.co/collect/zora:0xf898d4c188b1a742ef951d904a3f5c24e53d725a/4 which are currently open as of writing

https://api.zora.co/discover/contract/ZORA-MAINNET/0x44ac03c9101257fbd4f489d1ce9adcd466c1c27e?token_id=2

https://api.zora.co/discover/contract/ZORA-MAINNET/0xf898d4c188b1a742ef951d904a3f5c24e53d725a?token_id=4

### Functionality Supported
* Ingesting from URL: Yes / No
* Ingesting from Contract address: Yes / No
* Supported Networks: Base / Ethereum / Arbitrum / Zora / Polygon / Others..

### Before you submit
- [ ] Ensure your generated MintTemplate works 😄
- [ ] Ensure that your code is restricted to a single folder in `src/ingestors`
- [ ] Ensure that all required assets are included (e.g. ABIs)
- [ ] Ensure ABIs are trimmed to include only methods (1) used in the ingestor or (2) required to mint
- [ ] Ensure that all exported methods are prefixed with the name of your ingestor e.g. `myMintingPlatformGetContractDetails`
- [ ] Ensure that a test exists for generating a MintTemplate that will always succeed
- [ ] Ensure that your code accesses **no external resources** except those passed in the `resources` object

___

### **PR Type**
enhancement

___

### **Description**
- Removed the use of sales strategies as a fallback for determining the end date in the Zora metadata provider.
- The logic now focuses on the start date and calculates the live date based on the start date or the current date.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Enhancement</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>zora-metadata.ts</strong><dd><code>Remove sales strategies fallback for end date in Zora metadata</code></dd></summary>
<hr>

src/ingestors/zora-internal/zora-metadata.ts

<li>Removed fallback logic for end date using sales strategies.<br> <li> Retained logic for determining start date.<br> <li> Adjusted live date calculation to use start date or current date.<br>

</details>

  </td>
  <td><a href="https://github.com/floornfts/mobile-minting/pull/61/files#diff-e74e24465edcb730cfc4906246c4e47d13bccbb8a8eb3a5daffbdfa7eb8ae2b2">+1/-8</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

File: src/ingestors/zora-internal/zora-metadata.ts [hunk 1]
```diff
@@ -61,14 +61,7 @@ export class ZoraMetadataProvider {
     if (fallbackStartDateValue) {
       startDate = new Date(parseInt(fallbackStartDateValue) * 1000);
     }
-
-    let fallbackEndDateValue =
```
Comment (chrismaddern) [inline src/ingestors/zora-internal/zora-metadata.ts:65]: Maybe we get the end date and only if it's a non-0 value consider it?

Many Zora mints do have end dates that we probably want to capture if available?

**Review by chrismaddern — APPROVED** [review]
Not sure it's the right approach, but also, not the end of the world either way so do with it what you will & approved

File: src/ingestors/zora-internal/zora-metadata.ts (hunk 1 above)
Comment (agans) [reply src/ingestors/zora-internal/zora-metadata.ts:65]: but do we have any evidence they're ever represented by the data on the sale strategies...?

Comment (chrismaddern) [reply src/ingestors/zora-internal/zora-metadata.ts:65]: I believe they were at some point (these ingested properly until a week or so ago afaik)

I'm not sure whether it's a change in the data layout or the prevailing strategy in play though.

**Pull request merged**